helpers/useProcessSectionComplete.ts

```diff
@@ -16,7 +16,8 @@ const useProcessSectionComplete = <
   S extends StoreNames<ProcessDatabaseSchema["schema"]>
 >(
   processRef: ProcessRef,
-  storeNames: S[]
+  storeNames: S[],
+  validator?: Function
```

Comment:
Can we give this a signature? I think we know what we expect the input and output to be.